test/integration/clientset_test.go

```diff
@@ -466,6 +466,9 @@ func testInstanceClient(sType server.StorageType, client servicecatalogclient.In
 			ServiceClassName: "service-class-name",
 			PlanName:         "plan-name",
 			Parameters:       &runtime.RawExtension{Raw: []byte(instanceParameter)},
+			AlphaSecretParameters: &v1.LocalObjectReference{
```

Comment:
This shouldn't be a valid Instance; see other comments about adding a validation.